deployment/init-workspace.sh

```diff
@@ -0,0 +1,18 @@
+SRC_DIR="src/"
```

Comment:
Can we make the source dir for installing dependencies one subfolder above, i.e. catkin_ws/src rather than catkin_ws/src/mav_ipp? Then we don't need all the dependencies in the .gitignore and the packages in the same root folder.

Another way of handling dependencies is .rosinstall, example: https://github.com/ethz-asl/rotors_simulator/blob/master/dependencies.rosinstall